app/models/project.rb

```diff
@@ -114,6 +114,13 @@ class Project < ActiveRecord::Base
   before_destroy :delete_all_members
   before_destroy :destroy_all_work_packages
 
+  # HACK: remove on upgrade to awesome_nested_set 3.
+  # Active Record 4 will, by default, order by id:
+  #   ORDER BY "projects"."id" ASC LIMIT 1
+  # this results in the following genre of errors on PostgreSQL:
+  #   …it is not contained in either an aggregate function or the GROUP BY clause
+  default_scope order('')
```

Comment:
@myabc can you please elaborate on the conditions for this being problematic? nested_set is configured to used the `:name` column for sorting, so why is it causing problems that you need to counter with a default_scope? And when issuing some commands on the console, the `name` column is used for sorting. So why?